gbols/fast-food
# #160484167 admin can update the status of an orders.

**What does this PR do?**
 It updates status of an order.

**Description of Task to be completed?**
-  Bug fixes and updates

**How should this be manually tested?**
- pull from this branch `ft-update-order-status-160484167` into a new branch.
- run `npm install` to install all the requirements.
- endpoint should look like this `http://localhost:3000/api/v1/orders`
- run npm test to run the test.

**What are the relevant pivotal tracker stories?**
#160484167 Admin should be able to update the status of a food order.

**Postman's Screenshot**
![screenshot from 2018-09-13 17-08-43](https://user-images.githubusercontent.com/27724644/45500963-d9ed5100-b777-11e8-8085-8c07a689b4a2.png)
![screenshot from 2018-09-13 17-08-57](https://user-images.githubusercontent.com/27724644/45500968-db1e7e00-b777-11e8-9941-343bcff4bcb4.png)


**Postman Screenshots**
